packages/components/src/styles/components/form/super-select.scss

```diff
@@ -0,0 +1,502 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// SUPER-SELECT
+
+// Copied from components/src/styles/components/tag.scss styles
+// we set a higher value than the line-height (~13px) to accommodate cases where the text wraps
+$hds-tag-border-radius: 50px; // This variable is repeated from tag.scss
+$hds-super-select-item-height: 36px;
+
+// Style overrides:
+
+.hds-form-super-select {
+  width: 100%;
+}
+
+.hds-form-super-select .ember-basic-dropdown {
+
+  // TRIGGER
+  .ember-basic-dropdown-trigger,
+  .ember-power-select-trigger {
+    display: flex;
+    align-items: center;
+    max-width: 100%;
+    min-height: $hds-super-select-item-height;
+    padding-top: 4px;
+    padding-right: calc(var(--token-form-control-padding) + 24px); // extra space for the icon
+    padding-bottom: 4px;
+    padding-left: 11px;
```

Comment:
Yes, this was intentional. Since the `padding-top` is `12px` for the list, it looked odd if we made the `padding-left` and `padding-right` `16px` for the search (looked lopsided imo), so instead made it `12px` all around.

Reply:
Additionally I wanted it to look independent of the list items. The footer follows this logic too (`12px` `padding-left` and `padding-right`)